content/guides/building/get-source-git.md

```diff
@@ -71,15 +76,17 @@ git config core.precomposeunicode true
 The `<login>@` is only needed if your currently logged in username doesn't match your `review.haiku-os.org` username.
 
 ```sh
-git clone "ssh://<login>@git.haiku-os.org/buildtools" && scp -p <login>@git.haiku-os.org:hooks/commit-msg "buildtools/.git/hooks/"
+git clone "ssh://<login>@git.haiku-os.org/buildtools"
+scp -p <login>@git.haiku-os.org:hooks/commit-msg "buildtools/.git/hooks/"
```

Comment:
this changes the behavior. The && makes sure the second command is run only if the first has worked.